test/e2e/parallel/proxy_test.go

```diff
@@ -365,11 +364,12 @@ func TestProxyFlow(t *testing.T) {
 				require.NoError(t, err)
 				err = proxyCl.Create(context.TODO(), appToCreate)
 
-				// then
```

Comment:
This comment block is removed as it is no longer valid. Maintaining history here might be confusing